tests/rptest/clients/rpk.py

```diff
@@ -1743,6 +1743,12 @@ def transform_from_json(loaded):
 
         return [transform_from_json(o) for o in loaded]
 
+    def pause_wasm(self, name):
+        self._run_wasm(["resume", name])
```

Comment:
don't have the full context but shouldn't this be `"pause"`?

Reply:
weird. I have the right thing locally. anyway good catch - this also suggests that the corresponding test case is not well-formed... release mode ducktape passed 😕

Reply:
fixed